cli/tests/unit/console_test.ts

```diff
@@ -347,6 +347,21 @@ unitTest(function consoleTestStringifyCircular(): void {
 });
 /* eslint-enable @typescript-eslint/explicit-function-return-type */
 
+unitTest(function consoleTestStringifyFunctionWithPrototypeRemoved(): void {
+  const f = function f() {};
+  Reflect.setPrototypeOf(f, null);
+  assertEquals(stringify(f), "[Function: f]");
+  const af = async function af() {};
+  Reflect.setPrototypeOf(af, null);
+  assertEquals(stringify(af), "[Function: af]");
+  const gf = function gf() {};
+  Reflect.setPrototypeOf(gf, null);
+  assertEquals(stringify(gf), "[Function: gf]");
+  const agf = function agf() {};
+  Reflect.setPrototypeOf(agf, null);
+  assertEquals(stringify(agf), "[Function: agf]");
```

Comment:
I know this is already merged, but was this intended to test generator functions and async generator functions? (`function* gf() {}` and `async function* agf() {}`)

Reply:
@solson good catch, could you open an issue about it?

Reply:
> `function* gf() {}` and `async function* agf() {}`

Are you saying they should have their own output? That's just declaration-side syntactic sugar.

Reply:
No, I'm saying they should be declared that way here in the test, as in `const gf = function* gf() {};` to match `async function af` above, presumably to test this "set prototype to `null`" manipulation on all 4 of `Function`, `AsyncFunction`, `GeneratorFunction`, and `AsyncGeneratorFunction`. Currently, `gf` and `agf` are just testing the same thing as `f`.

Reply:
@bartlomieju https://github.com/denoland/deno/issues/7962